includes/object-cache.php

```diff
@@ -2472,7 +2473,7 @@ public function switch_to_blog( $_blog_id ) {
             return false;
         }
 
-        $this->blog_prefix = $_blog_id;
+        $this->blog_prefix = (string) $_blog_id;
```

Comment:
I rather keep this a `(int)` across the codebase.

Reply:
Okay! Much better!
Than please change type of `blog_prefix` property.